docker/testnet/docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+version: "3.8"
+x-services-volume:
+  &services-volume
+  - type: volume
+    source: db
+    target: /opt/diem/var
+  - type: bind
+    source: ./root.key
+    target: /opt/diem/var/root.key
+services:
+    validator:
+        image: diem/validator:release-1.1_ab1761a8
```

Comment:
the problem is that this version is ephemeral that's why we chose testnet. and we chose libra because it doesn't have those tags. I need to harass my dev-infra friends ;)

Reply:
alright, switch later when premainnet tag is available.